UI/window-projector.hpp

```diff
@@ -49,6 +49,13 @@ class OBSProjector : public OBSQTDisplay {
 	gs_vertbuffer_t *rightLine = nullptr;
 	bool ready = false;
 
+	// argb colors
+	static const uint32_t outerColor     = 0xFFD0D0D0;
```

Comment:
Indent here doesn't appear to be the same as the ones below it.  Just a totally minor and insignificant nitpick.